<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManufacturerWindow</class>
 <widget class="QMainWindow" name="ManufacturerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRoles">
    <property name="title">
     <string>Roles</string>
    </property>
    <addaction name="actionSwitch_Role"/>
   </widget>
   <widget class="QMenu" name="menuJobs">
    <property name="title">
     <string>Jobs</string>
    </property>
    <addaction name="actionJobs_List"/>
   </widget>
   <addaction name="menuRoles"/>
   <addaction name="menuJobs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSwitch_Role">
   <property name="text">
    <string>Switch Role</string>
   </property>
  </action>
  <action name="actionJobs_List">
   <property name="text">
    <string>Jobs List</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
